source/Registry.h

```diff
@@ -63,6 +65,7 @@ class Registry final {
   /* These are thread-safe. */
   Model get(const Method* method) const;
   FieldModel get(const Field* field) const;
+  LiteralModel get(std::string_view literal) const;
```

Comment:
Could we change the name of this method?
When I read `get` I usually expect a `O(1)` but here we loop over all literal models.
Also, we might want another API latter that is an actual `get` for a given pattern.
I would call it `match_literal` or something.
Also nitpick: the comment above says "These are thread-safe" but your implementation of `get` is not thread-safe (iterating on a ConcurrentMap is not thread safe). I think it's fine though as long as there are no mutations in parallel.

Reply:
Technically not my comment, but definitely surprised to learn that redex' `ConcurrentMap` concurrent traversal isn't defined. Not what I'm used to from `concurrent_unordered_map`. 🙂 

Since I renamed the method, I now moved it away from those two `get` mehods and gave it a comment of its own.